packages/nimble-components/src/icon-base/icon-metadata.ts

```diff
@@ -214,6 +214,9 @@ export const iconMetadata: {
     IconExclamationMark: {
         tags: ['error', 'warning']
     },
+    IconExport: {
```

Comment:
@NIbokeefe we already had a share icon that looks similar to the new export icon (but not identical). Do we want to keep them both? Any guidance on when to use one versus the other?

<img width="1105" alt="image" src="https://github.com/ni/nimble/assets/10500124/3a69484b-d01b-43e7-be41-e31c823e543d">